app/src/main/res/layout/login_fragment.xml

```diff
@@ -3,124 +3,114 @@
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="0.043" />
+        app:layout_constraintTop_toTopOf="parent" />
 
     <Button
-        android:id="@+id/btnGoogle"
-        android:layout_width="match_parent"
+        android:id="@+id/btnSkip"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_margin="@dimen/size20"
-        android:text="@string/google"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.518"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/tvGood" />
+        android:layout_marginLeft="@dimen/margin12"
+        android:layout_marginTop="@dimen/margin52"
+        android:layout_marginRight="@dimen/margin12"
+        android:text="@string/skip"
+        android:textColor="@color/black"
+        android:textSize="@dimen/text10"
+        app:layout_constraintStart_toEndOf="@+id/btnHindi"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
 
     <TextView
-        android:id="@+id/tvOr"
+        android:id="@+id/tvLogin"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_margin="@dimen/size30"
-        android:text="@string/or"
+        android:layout_marginTop="@dimen/margin26"
+        android:layout_marginLeft="@dimen/margin16"
+
+        android:text="@string/Login"
         android:textColor="@color/black"
         android:textSize="@dimen/text20"
-        app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/btnGoogle" />
+        app:layout_constraintTop_toBottomOf="@+id/btnHindi"
+        android:layout_marginStart="@dimen/margin16" />
 
-    <EditText
+    <com.google.android.material.textfield.TextInputLayout
         android:id="@+id/editPhone"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_margin="@dimen/size30"
-        android:ems="10"
-        android:textColorHint="@color/black"
-        android:textColor="@color/black"
-        android:hint="@string/phone"
-
-        android:inputType="phone"
-        app:layout_constraintEnd_toEndOf="@+id/btnGoogle"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/tvOr" />
-
-    <Button
-        android:id="@+id/btnLogin"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_margin="@dimen/size30"
-        android:text="@string/login"
+        android:layout_marginTop="@dimen/margin99"
+        android:layout_marginLeft="@dimen/margin16"
+        android:layout_marginRight="@dimen/margin16"
+        app:boxBackgroundColor="@color/azure"
```

Comment:
@raghav32 as per the design the background of the editText field is white. But as per this, you are showing blue background, please fix this